Source/Core/Core/PowerPC/PowerPC.h

```diff
@@ -288,16 +288,19 @@ struct TranslateResult
 };
 TranslateResult JitCache_TranslateAddress(u32 address);
 
-static const int BAT_INDEX_SHIFT = 17;
+constexpr int BAT_INDEX_SHIFT = 17;
```

Comment:
I can't see how you could achieve similar result using ```x * size```. How do you plan to use it?

I don't mind keeping the ```x << BAT_INDEX_SHIFT ``` or ```x >> BAT_INDEX_SHIFT ```. Do you think it would be easier for other people to understand if we replace them with ```SetBatIndex``` and ```GetBatIndex``` functions?